genesis-tools/lockup-contract/src/lib.rs

```diff
@@ -0,0 +1,220 @@
+//! A smart contract that allows lockup of a smart contract.
+use borsh::{BorshDeserialize, BorshSerialize};
+use key_management::{KeyType, PublicKey};
+use near_bindgen::{env, near_bindgen, Promise};
+use std::collections::{HashMap, HashSet};
+
+mod key_management;
+
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+
+#[near_bindgen]
+#[derive(Default, BorshDeserialize, BorshSerialize)]
```

Comment:
Remove Default, so the contract can't be called without init.

Reply:
Done